org.opent2t.sample.hub.superpopular/com.insteon.hub/js/package.json

```diff
@@ -21,8 +21,10 @@
   "dependencies": {
     "request": "^2.74.0",
     "request-promise": "^4.1.1",
-    "opent2t": "^1.0.1",
-    "opent2t-onboarding-org-opent2t-onboarding-insteon": "^0.0.0",
+    "opent2t": "^1.0.2",
+    "opent2t-onboarding-org-opent2t-onboarding-insteon": "^0.2.0",
```

Comment:
Checked with Avneet that we will be in 0.2.x for the latest breaking change. This translator will be using the a new version of onborder (in PR), which will be version 0.2.0. Please let me if otherwise.

---
In reply to: [98254026](https://github.com/openT2T/translators/pull/149#discussion_r98254026) [](ancestors = 98254026)